src/components/rangeUsePlanPage/versionsList/VersionsDropdownList.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import Box from '@material-ui/core/Box'
+import Collapse from '@material-ui/core/Collapse'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import moment from 'moment'
+import classnames from 'classnames'
+import Status from '../../common/Status'
+
+const VersionsDropDownList = ({ versions, open }) => {
+  const versionOptions = versions.map(v => ({
+    key: v.version,
+    value: v,
+    text: `v${v.version}`,
+    content: (
```

Comment:
This `content` property isn't used anywhere and can probably be removed. I think you copied this from the other version component, where I had accidentally left it in. That's my mistake.

Reply:
Removed in this commit: e0941af